### ISSUE TITLE
[bug report] crash because of a missing `style.sign` override

### ISSUE THREAD

**Issue by Picalines** [everviolet/nvim#8]
There's a change in the recent [commit](https://github.com/comfysage/evergarden/commit/dddaf99b9ef16f7046710f905d819704bc0e5599) that introduced an issue. It assumes that there's a `sign` field in `style` configuration

https://github.com/comfysage/evergarden/blob/dddaf99b9ef16f7046710f905d819704bc0e5599/lua/evergarden/theme.lua#L83

which wasn't the case for my configuration:

```lua
opts = {
  transparent_background = false,
  contrast_dark = 'hard',
  override_terminal = true,
  style = {
	  tabline = { reverse = true, color = 'green' },
	  search = { reverse = false, inc_reverse = true },
	  types = { italic = true },
	  keyword = { italic = true },
	  comment = { italic = true },
          -- no sign!
  },
  overrides = {},
},
```

and that's why I get the error:

```
E5113: Error while calling lua chunk: ...lua/evergarden/theme.lua:83: attempt to index field 'sign' (a nil value)
```

The solution is obviously to add the missing `sign` option. But it seems to me that at the plugin level, you can add a deep merge of configurations so that such problems do not arise

sidenote: one of the best color themes, thank you very much!

**Comment by tan-wei** [comment]
Met the same issue.

**Comment by comfysage** [comment]
you're absolutely right. seems i forgot to stash some changes before committing.

**Comment by comfysage** [comment]
should be fixed with 7e031b6 - sorry for the issues
